vendor/github.com/jesseduffield/gocui/tcell_driver.go

```diff
@@ -279,17 +279,11 @@ func (g *Gui) pollEvent() GocuiEvent {
 			mod = 0
 			ch = rune(0)
 			k = tcell.KeyCtrlSpace
-		} else if mod == tcell.ModCtrl || mod == tcell.ModShift {
+		} else if mod == tcell.ModShift {
 			// remove Ctrl or Shift if specified
```

Comment:
we can remove mention of ctrl in this comment now

Reply:
Also we should probably let ModShit flow through.